tests/instancetype/instancetype.go

```diff
@@ -637,18 +630,18 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 			Expect(err).ToNot(HaveOccurred())
 
 			By("Creating a VirtualMachine")
-			vm := libvmi.NewVirtualMachine(vmi, libvmi.WithInstancetype(instancetype.Name))
+			vm := libvmi.NewVirtualMachine(vmi, libvmi.WithInstancetype(instancetype.Name), libvmi.WithRunStrategy(virtv1.RunStrategyAlways))
 			vm, err = virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Create(context.Background(), vm, metav1.CreateOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
-			vm = libvmops.StartVirtualMachine(vm)
+			By("Waiting for VM to be ready")
+			EventuallyWithOffset(1, matcher.ThisVM(vm), 360*time.Second, 1*time.Second).Should(matcher.BeReady())
+			vm, err = virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Get(context.Background(), vm.Name, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
 
-			By("Waiting for VirtualMachineInstancetypeSpec ControllerRevision to be referenced from the VirtualMachine")
-			Eventually(func(g Gomega) {
-				vm, err = virtClient.VirtualMachine(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
-				g.Expect(err).ToNot(HaveOccurred())
-				g.Expect(vm.Spec.Instancetype.RevisionName).ToNot(BeEmpty())
-			}, 5*time.Minute, time.Second).Should(Succeed())
+			By("VirtualMachineInstancetypeSpec ControllerRevision should be referenced from the VirtualMachine")
+			Expect(err).ToNot(HaveOccurred())
+			Expect(vm.Spec.Instancetype.RevisionName).To(ContainSubstring(instancetype.Name))
```

Comment:
:+1:

Reply:
I'm not sure i understand

Reply:
My point was just that the lookup and assertion code you've added here could instead use a common `Eventually` call and would result in the same number of API requests.

Reply:
Please correct me if I'm wrong, but there's no need for eventually here. At this point, the VM is ready, and the revisionName shouldn't be ready.